src/style.css

```diff
@@ -0,0 +1,42 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars__container {
+  display: flex;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: 50%;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--5 .stars__star,
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 .stars__star:not(:first-child),
+.stars--2 .stars__star:nth-child(n+3),
+.stars--3 .stars__star:nth-child(n+4),
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url(./images/star.svg);
+}
```

Comment:
you don't have to write selectors for inactive stars, add them in stars__star, and selectors with star-active will rewrite them where needed